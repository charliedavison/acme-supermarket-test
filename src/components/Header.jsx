import React from 'react';
import PropTypes from 'prop-types';

class Header extends React.Component {
    constructor(props) {
        super(props);
    };

    render() {
        return (
            <div className="header">
                <h1>ACME Supermarket</h1>
            </div>
        )
    }
};
Header.defaultProps = {
    basketItemCount: 0
};
Header.propTypes = {
    basketItemCount: PropTypes.number.isRequired
};
module.exports = Header;
