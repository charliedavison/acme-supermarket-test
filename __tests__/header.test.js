import React from 'react';
import { shallow } from 'enzyme';
import Header from './../src/components/Header.jsx';

describe('<Header />', () => {
    test('Should match snapshot', () => {
        const component = shallow(<Header />);
        expect(component).toMatchSnapshot();
    });
});