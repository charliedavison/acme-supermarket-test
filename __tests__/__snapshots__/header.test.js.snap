// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Should match snapshot 1`] = `
<div
  className="header"
>
  <h1>
    ACME Supermarket
  </h1>
</div>
`;
